business/cacheRefresher.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const DefinitionsCache = require('./definitionsCache')
+const utils = require('../lib/utils')
+const _ = require('lodash')
+
+class CacheRefresher {
+  constructor(harvest, curation) {
+    this.harvestService = harvest
+    this.curationService = curation
+    setInterval(this.refreshDefinitions.bind(this), 4 * 60 * 1000)
```

Comment:
If `refreshDefinitions` fails for any reason then it will take down the process, I'd wrap it in another error safe function that logs any errors but suppresses them,